diesel/src/query_builder/upsert/on_conflict_actions.rs

```diff
@@ -89,10 +130,11 @@ where
     }
 }
 
-impl<DB, T, SP> QueryFragment<DB, SP> for Excluded<T>
+impl<DB, T>
+    QueryFragment<DB, crate::backend::sql_dialect::on_conflict_clause::PgLikeOnConflictClause>
```

Comment:
Again, breaks sqlite.

Reply:
Same question re: what is meant by "break" since it still compiles (for me, at least!)